ossm/vendor/envoy/bssl-compat/prefixer/prefixer.cpp

```diff
@@ -112,7 +112,7 @@ class Function {
 
     std::string getHeader(const clang::SourceManager &srcmgr) const {
       clang::SourceLocation sloc = getFileLoc(srcmgr, m_node->getLocation());
-      const clang::FileEntry *declfile = srcmgr.getFileEntryForID(srcmgr.getFileID(sloc));
+      const clang::OptionalFileEntryRef declfile = srcmgr.getFileEntryRefForID(srcmgr.getFileID(sloc));
```

Comment:
Don't we wan to change this in envoy-openssl repo instead of here?

Reply:
🤦🏻‍♂️